tests/Integration/Integrations/PredisTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace DDTrace\Tests\Integration\Integrations;
+
+use DDTrace\Encoders\Json;
+// use DDTrace\Tests\RequestReplayer;
+use DDTrace\Tracer;
+use DDTrace\Transport\Http;
+// use DDTrace\Version;
+use DDTrace\Integrations\Predis;
```

Comment:
Yeah sorting alphabetically sounds like fun :) (if it can be done automatically)

However I'd much prefer to have `php-cs-fixer`  do it. It should be possible, lets add that in separate PR, as it might trigger reformat in the rest of the code.